src/app/layout.tsx

```diff
@@ -14,8 +14,9 @@ import { SessionProvider } from "../contextProviders/session";
 import { authOptions } from "./api/utils/auth";
 import { metropolis } from "./fonts/Metropolis/metropolis";
 import { CONST_GA4_MEASUREMENT_ID } from "../constants";
-import { GoogleAnalytics } from "@next/third-parties/google";
 import { headers } from "next/headers";
+import { GoogleAnalyticsWorkaround } from "./components/client/GoogleAnalyticsWorkaround";
+import { getNonce } from "./deprecated/functions/server/getNonce";
```

Comment:
Should we use `const nonce = headers().get("x-nonce") ?? ""` to get `nonce` like we do [here](https://github.com/mozilla/blurts-server/pull/4161/files#diff-521160afb661a91142c95654d4bf5644f90993f401587bf71f4b5871e26662e6R36)?